setup.py

```diff
@@ -97,7 +97,7 @@
     name="pyautogen",
     version=__version__,
     author="AutoGen",
-    author_email="auto-gen@outlook.com",
+    author_email="autogen-contact@service.microsoft.com",
```

Comment:
Would like to know why the changes are needed.

Reply:
Hi Qingyun, none of the other maintainers have access to this private outlook.com address. From a security and transparency standpoint microsoft/autogen needs a better distribution list. You are are already on this new address.